pages/PoojaList.tsx

```diff
@@ -1,18 +1,50 @@
-import React, { Fragment } from 'react'
+import { deleteCookie, getCookie } from 'cookies-next';
+import { useRouter } from 'next/router';
+import React, { Fragment, useEffect, useState } from 'react'
 import Pooja from './PoojaCard'
 
-import { DUMMY_POOJAS } from '../public/data'
-
 const PoojaList = () => {
-  const poojaList = Object.values(DUMMY_POOJAS).map((pooja) => (
-    <Pooja
-      key={pooja.id}
-      id={pooja.id}
-      name={pooja.name}
-      imageUrl={pooja.imageUrl}
-      description={pooja.description}
-    />
-  ))
+  const router = useRouter();
+
+  const [poojaList, setPoojaList] = useState([]);
+
+  useEffect(() => {
+    const getAllPooja = async () => {
+      const xBhaktToken = getCookie("xBhaktToken") + "";
+      const response = await fetch('/api/pooja', {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          'x-bhakt-token': xBhaktToken
+        }
+      });
+      console.log(response);
+      const responseJson = await response.json();
+      console.log(responseJson);
+
+      if (response.status === 200) {
+        const poojaData = responseJson["allPooja"].map((pooja: { id: number; name: string; imageUrl: string; description: string; } ) => (
+          <Pooja
+            key={pooja.id}
+            id={pooja.id}
+            name={pooja.name}
+            imageUrl={pooja.imageUrl}
+            description={pooja.description}
+          />
+        ))
+        setPoojaList(poojaData);
+      }
+      else if (response.status === 401) {
+        deleteCookie('xBhaktToken')
+        router.push('/Login')
+      }
+      else {
+        alert(responseJson.message)
+      }
+    }
+    getAllPooja();
+  }, []);
+
```

Comment:
Please add 2nd case mentioned in the issue
`It should redirect to login if token absent in cookies`. I know its working but its due to API responding with invalid token